src/io/io.h

```diff
@@ -0,0 +1,30 @@
+#ifndef INCLUDE_IO_H
+#define INCLUDE_IO_H
+/** outb:
+* Sends the given data to the given I/O port. Defined in io.s
+*
+* @param port The I/O port to send the data to
+* @param data The data to send to the I/O port
+25
+*/
+extern "C" void outb(unsigned short port, unsigned char data);
+
+/* The I/O ports */
+#define FB_COMMAND_PORT 0x3D4
+#define FB_DATA_PORT 0x3D5
+/* The I/O port commands */
+#define FB_HIGH_BYTE_COMMAND 14
+#define FB_LOW_BYTE_COMMAND 15
+/** fb_move_cursor:
+* Moves the cursor of the framebuffer to the given position
+*
+* @param pos The new position of the cursor
+*/
+void fb_move_cursor(unsigned short pos) {
```

Comment:
You said `fb` stands for frame buffer, right? Can we make this more clear by renaming the function? I think verbosity will probably do us more good than harm since there will be a lot of new code that neither of us have seen before!

Reply:
Done.